mir_eval/sonify.py

```diff
@@ -131,12 +131,18 @@ def _fast_synthesize(frequency):
         # Get a waveform of length samples at this frequency
         wave = _fast_synthesize(frequency)
         # Scale each time interval by the piano roll magnitude
-        for m, (start, end) in enumerate((times * fs).astype(int)):
+        for m, (start, end) in enumerate((times[:-1, :] * fs).astype(int)):
             # Clip the timings to make sure the indices are valid
             start, end = max(start, 0), min(end, length)
+            hop_size = end - start
+            xover_size = int(hop_size / 8)
```

Comment:
The 8 should probably be user-settable, unless we are sure this will work well in all the cases we care about.  I am concerned, for example, that if `times` had any really long-duration intervals, it would sound kind of over-the-top.